show-all.md

```diff
@@ -24,6 +24,9 @@ Variable | Description
 ------|------------
 `DATABASE` | The default database for the current session, as set by [`SET DATABASE`](set-database.html) or the client's connection string. This variable can be viewed with [`SHOW DATABASE`](show-database.html) as well. 
 `DEFAULT_TRANSACTION_ISOLATION` | The default transaction isolation level for the current session, as set by `SET DEFAULT_TRANSACTION_ISOLATION` or the client's connection string. 
+`MAX_INDEX_KEYS` | Not usable; exposed only for ORM compatibility.
+`SEARCH_PATH` | A list of databases or name spaces that will be searched to resolve unqualified table or function names. This variable can be changed with `SET SEARCH_PATH`.
```

Comment:
Namespaces should be a single word.

Also, we don't actually support changing this with `SET`, so I would drop the last sentence.